From c6b11bfdef50b338eecc9a933b531a03b06d8e1e Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Thu, 25 May 2017 20:40:41 +0800
Subject: [PATCH] ARM: rockchip: pm: fix compile warning

When build with ARM_LPAE enabled:
arch/arm/mach-rockchip/pm.c: In function 'rk3288_init_pmu_sram':
arch/arm/mach-rockchip/pm.c:85:23: warning: cast to pointer from
integer of different size [-Wint-to-pointer-cast]

Change-Id: I526f13439c3c7edde77a27419d488a8f79081661
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm/mach-rockchip/pm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-rockchip/pm.c b/arch/arm/mach-rockchip/pm.c
index 8d956f7a13e9..03646f7890d1 100644
--- a/arch/arm/mach-rockchip/pm.c
+++ b/arch/arm/mach-rockchip/pm.c
@@ -82,7 +82,7 @@ static void __init rk3288_init_pmu_sram(void)
 	/* setup the params that we know at boot time */
 	params = (struct rk3288_resume_params *)rk3288_bootram_base;
 
-	params->cpu_resume = (void *)virt_to_phys(cpu_resume);
+	params->cpu_resume = (void *)(u32)virt_to_phys(cpu_resume);
 
 	params->l2ctlr_f = 1;
 	params->l2ctlr = rk3288_l2_config();
-- 
2.35.3

